js/scoreFormatter.js

```diff
@@ -110,3 +120,39 @@ YoastSEO.ScoreFormatter.prototype.scoreRating = function( score ) {
 	}
 	return scoreRate;
 };
+
+/**
+ * Returns a translated score description based on the textual score rating
+ *
+ * @param {string} scoreRating Textual score rating, can be retrieved with scoreRating from the actual score.
+ * @param {Jed}    i18n A translation object to use when translating the strings.
+ *
+ * @return {string}
+ */
+YoastSEO.ScoreFormatter.prototype.getSEOScoreText = function( scoreRating, i18n ) {
+	var scoreText = "";
+
+	switch ( scoreRating ) {
+		case "na":
+			scoreText = i18n.dgettext( "js-text-analysis", "No keyword" );
+			break;
```

Comment:
na is used for when the score is not available. Currently we use this only when no keyword is specified, but this could change in de future.